oci-artificial-intelligence/forecasting-service/item-4-forecasting-service-create-ds-notebook/item-4-forecasting-service-create-ds-notebook.md

```diff
@@ -94,37 +95,35 @@ In this lab, we will learn how to Forecast using Data Science Notebook. We will
 
     ```
 
-    ![](images/lab1-task3-step9-authorization.png " ")
+    ![Authorization](images/lab1-task3-step9-authorization.png " ")
 
 
 ## Task 4: Understand Data, Download Samples, Prepare data, Create Project
 OCI forecasting service provides an AutoML solution with multiple univariate/multivariate algorithms that can run on single series or multiple series at once. There are some data validations and data format requirements that the input data must satisfy.
 
 **Data Validations**
 
-For a successful forecast, the input data should pass the following data validations:
+FFor a successful forecast, the input data should pass the following data validations:
```

Comment:
Hi Ravijeet, it looks like this "FFor" is a typo

Reply:
Thanks for reviewing . Yes, it's a typo.  Fixed it.